server/app/api/models/project_allocation.js

```diff
@@ -3,31 +3,33 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
```

Comment:
camelcase schema

Reply:
This is syntax for Schema. Uppercase 'S'